### DISCUSSION TITLE
Action to create does not load the defined label

### DISCUSSION THREAD

**Discussion by orebarranco** [saade/filament-fullcalendar#182] [Q&A] [answered]
I have the Appointment model defined. I have set the modelLabel property to a value. But the create button does not show the label that I have set for the model. It simply shows the model name.
Am I missing something to define, or is it a package error?

![image](https://github.com/saade/filament-fullcalendar/assets/106551888/6b3585f2-2e59-44a1-bb09-f16038a3ab24)

![image](https://github.com/saade/filament-fullcalendar/assets/106551888/64547078-2a67-4073-8bfb-82614830e74a)

**Comment by jncalderon** [comment]
@orebarranco Hi do you have some solution for it?

**Reply by orebarranco** [reply, marked as answer]
Hi, I solved the problem by overriding the headerActions() method and specifying the label method for the CreateAction action:
```
protected function headerActions(): array
    {
        return [
            \Saade\FilamentFullCalendar\Actions\CreateAction::make()
                ->label('Crear Cita')
                ->url(AppointmentResource::getUrl('create')),
        ];
    }
```

**Reply by jncalderon** [reply]
thanks men, make sense!!

**Comment by jncalderon** [comment]
@saade any idea for this issue? even I see the trait InteractsWithRecords doesn't work
